packages/evo-blockmodels/src/evo/blockmodels/endpoints/models.py

```diff
@@ -510,13 +587,21 @@ class RotationAxis(Enum):
 
 
 class Size3D(CustomBaseModel):
-    nx: StrictInt = Field(..., title="Nx")
-    ny: StrictInt = Field(..., title="Ny")
-    nz: StrictInt = Field(..., title="Nz")
+    nx: Annotated[StrictInt, Field(title="Nx")]
+    ny: Annotated[StrictInt, Field(title="Ny")]
+    nz: Annotated[StrictInt, Field(title="Nz")]
+
+
+class ModelType(Enum):
+    """
+    Type of sub-blocking
+    """
+
+    flexible = "flexible"
 
 
 class SizeOptionsFlexible(CustomBaseModel):
-    model_type: Literal["flexible"] = Field(title="Model Type")
+    model_type: Annotated[Literal["flexible"], Field(title="Model Type")] = "flexible"
```

Comment:
For `evo-blockmodels` in particular, these default values were recently removed due to an unintended interaction with the serialization logic. See #124 for more details

```suggestion
    model_type: Annotated[Literal["flexible"], Field(title="Model Type")]
```